docs/develop/java/observability.mdx

```diff
@@ -171,54 +171,83 @@ Here is how to query Workflow Executions:
 
 After you've created custom Search Attributes in your Temporal Service (using `temporal operator search-attribute create` or the Cloud UI), you can set the values of the custom Search Attributes when starting a Workflow.
 
-To set a custom Search Attribute, call the [`setSearchAttributes()`](https://www.javadoc.io/doc/io.temporal/temporal-sdk/latest/io/temporal/client/WorkflowOptions.Builder.html#setSearchAttributes(java.util.Map)) method.
+When starting a Workflow execution with your Client, include the Custom Search 
+Attribute in the options using `WorkflowOptions.newBuilder().setTypedSearchAttributes()`:
```

Comment:
This may be a "old regime" comment and we don't care under new regime, but sentence split across two lines.